src/main/java/core/basesyntax/FileEditor.java

```diff
@@ -0,0 +1,233 @@
+            switch (inputWords[0].toLowerCase()) {
+                case ("create") :
+                    createFile(inputWords[1],inputWords[2]);
+                    break;
+                case ("read") :
+                    readFile(inputWords[1], inputWords[2]);
+                    break;
+                case ("info") :
+                    fileInfo(inputWords[1], inputWords[2]);
+                    break;
+                default :
+                    writeTextToFile(input);
+                    break;
+            }
+        } else {
+            writeTextToFile(input);
+        }
+
+    }
+
+    private void createFile(String filepath, String filename) {
+        Path directoryPath = Paths.get(filepath);
+        Path filePath = Paths.get(filepath + File.separator + filename + ".txt");
+        if (!Files.exists(directoryPath)) {
+            System.out.println("This directory does not exist. Create this directory? (Y/N)");
+            String answer = scanner.nextLine();
+            if (answer.toLowerCase().equals("y")) {
+                try {
+                    Files.createDirectory(directoryPath);
+                } catch (IOException e) {
+                    System.out.println("Problem with directory creation");
+                    returnToBeginning();
+                }
+            } else {
+                returnToBeginning();
+            }
+        }
+        if (Files.exists(filePath)) {
+            System.out.println("File is already exists. Override file? (Y/N)");
+            String answer = scanner.nextLine();
+            if (answer.toLowerCase().equals("y")) {
+                try {
+                    Files.delete(filePath);
+                    Files.createFile(filePath);
+                } catch (IOException e) {
+                    System.out.println("Some problems with overriding");
+                    returnToBeginning();
+                }
+                System.out.println("File successfully created!");
+            } else {
+                returnToBeginning();
+            }
+        } else {
+            try {
+                Files.createFile(filePath);
+                returnToBeginning();
+            } catch (IOException e) {
+                System.out.println("Some problems with file creation");
+            }
+            System.out.println("File successfully created!");
+        }
+
+    }
+
+    private void readFile(String filepath, String filename) {
+        Path filePath = Paths.get(filepath + File.separator + filename + ".txt");
+
+        try {
+            List<String> lines = Files.readAllLines(filePath);
+            for (String line : lines) {
+                System.out.println(line);
+            }
+            returnToBeginning();
+        } catch (IOException e) {
+            System.out.println("No such file or directory");
+            returnToBeginning();
+        }
+
+    }
+
+    private void fileInfo(String filepath, String filename) {
```

Comment:
```suggestion
    private void getFileInfo(String filepath, String filename) {
```
https://mate-academy.github.io/style-guides/java/java.html#s5.2.3-method-names
`Method names are typically verbs or verb phrases. For example, sendMessage or stop.`